### ISSUE TITLE
Function to_datetime cannot handle some formats 

### ISSUE THREAD

**Issue by pguso** [opensource9ja/danfojs#267]
**Describe the bug**
If the date string contains one of the following symbols / . _ : or %y the function to_datetime throws an error "Invalid date, the date format not recognise"

**To Reproduce**
The following code throws the mentioned error.

````
import * as dfd from "danfojs-node"

// Example with dot
const data = new dfd.Series([ "12.30.19", "12.22.19", "12.01.19" ]);
const times = dfd.to_datetime({ "data": data, "format": "%m.%d.%y" });

// Example with / : and %y
const other_data = new dfd.Series([ "12/30/19 00:01", "12/29/19 07:03", "12/12/19 18:21" ]);
const other_times = dfd.to_datetime({ "data": other_data, "format": "%m/%d/%y %H:%M" });
````

**Expected behavior**
A TimeSeries object that can be used for further analysis.

**Desktop (please complete the following information):**
 - OS: MacOS
 - Node Version: v14.17.5

**Comment by risenW** [comment]
Noted, currently working on a fix

**Comment by risenW** [comment]
Fixed in #235
